magprime/templates/regextra.html

```diff
@@ -24,15 +24,18 @@
             $('#guitar_keychain,#tax_exempt').empty();
         }
         {% if c.PAGE_PATH != '/registration/form' %}
+            // TODO: rip this hardcoded stuff out?
             if ($.field('amount_extra')) {
                 $.field('amount_extra').parents('.form-group').append($('#tax_exempt'));
                 $("input:radio[name='amount_extra'][value=40]").parents('.btn').append($('#guitar_keychain'));
             }
+
             if ($.field('shirt')) {
```

Comment:
This block can stay, assuming Merch still only sells the 'slim fit' sizes as pre-orders (which is what the div this code places/hides explains).